scripts/build_utils.sh

```diff
@@ -855,6 +856,10 @@ ceph_build_args_from_flavor() {
         CEPH_EXTRA_CMAKE_ARGS+=" -DCMAKE_BUILD_TYPE=Debug"
         CEPH_EXTRA_CMAKE_ARGS+=" -DWITH_SEASTAR=ON"
         ;;
+    jaeger)
+	CEPH_EXTRA_RPMBUILD_ARGS="--with jaeger"
+	export PROFILES="pkg.ceph.jaeger"
```

Comment:
could you please use space for indent? there is no need to use `export` here, i think.

Reply:
ack, done